src/subdomains/generic/user/models/wallet/dto/create-wallet.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { IsEnum, IsNotEmpty } from 'class-validator';
+import { AmlRule } from '../wallet.entity';
+import { WalletDto } from './wallet.dto';
+
+export class CreateWalletDto extends WalletDto {
+  @IsNotEmpty()
+  @IsEnum(AmlRule)
+  amlRule: AmlRule;
+}
```

Comment:
Why is this different from WalletDto and UpdateWalletDto?

Reply:
Hehe ups, good question 😁 
didn't see the default